encode/av1encode.c

```diff
@@ -25,6 +25,12 @@
         exit(1);                                                        \
     }
 
+#define CHECK_FN(cod, func)                                                     \
+    if (!cod) {                                                                 \
```

Comment:
Add brackets around cond:
```
if (!(cond))
```
Otherwise you will get wrong substitution:
```
CHECK_FN(ret) -> if (!ret < 0)
```
while it should be `if (!(ret < 0))`.

Reply:
got it, thanks~